packages/lib-react-components/src/ProjectCard/components/CardHeader.js

```diff
@@ -0,0 +1,82 @@
+import { Box, Text } from 'grommet'
+import { number, string } from 'prop-types'
+import styled from 'styled-components'
+
+import { useTranslation } from '../../translations/i18n'
+import SpacedText from '../../SpacedText'
+
+const StyledBox = styled(Box)`
+  background: ${props => props.state === 'paused' ? 
+    'rgba(67, 187, 253, 0.5)'
+    : props.state === 'finished' ?
+    'rgba(228, 89, 80, 0.5)'
+    : undefined};
+  position: relative;
+`
+
+const StyledBadge = styled(Text)`
+  background: white;
+  border-radius: 0.5rem;
+  box-shadow: 1px 1px 4px 0px rgba(0, 0, 0, 0.25);
+  height: fit-content;
+  min-width: 0.75rem;
+  padding: 2px 3px;
+  position: absolute;
+  right: 5px;
+  text-align: center;
+  top: 5px;
+`
+
+function CardHeader({
+  badge,
+  state = 'live'
+}) {
+  if ((state === 'live' || state === 'development') && !badge) return null
```

Comment:
I'm thinking this needs a comment noting the different values that `state` can be and changed to: 
```suggestion
  // project.state can be 'live', 'paused', 'finished', or 'development'
  if ((state !== 'paused' || state !== 'finished') && !badge) return null
```

Reply:
Good idea! I remember that in a project's admin page there's also a radio button for "active" in the same toggle as "paused" and "finished". Does that have any influence on the UI here?

Reply:
Good question, I wasn't sure about "active" and had to check - confirmed it [changes the project state to "live"](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/pages/admin/project-status.jsx#L469) so I these conditionals should capture all the project states.